README.md

```diff
@@ -25,26 +25,45 @@ To build, run `mvn clean install`.
 | master  | 0.6.0-SNAPSHOT | 0.11.0-SNAPSHOT         |
 
 
-## Example
```

Comment:
I can't leave a comment against the line but you can delete the note above saying "However, it currently depends on the 10MB aws-sdk so this dependency has deliberately not been added to core brooklyn."